lib/taurus/qt/qtgui/qwt5/cli.py

```diff
@@ -58,34 +72,22 @@ def plot_cmd(models, config_file, x_axis_mode, demo, window_name):
 
 
 @qwt5.command('trend')
-@click.argument('models', nargs=-1)
-@click.option("-x", "--x-axis-mode", "x_axis_mode",
-              type=click.Choice(['t', 'n']),
-              default='n',
-              show_default=True,
-              help=('X axis mode. "t" implies using a Date axis'
-                    + '"n" uses the regular axis')
-              )
+@taurus.cli.common.models
+@x_axis_mode
+@taurus.cli.common.config_file
+@taurus.cli.common.demo
+@taurus.cli.common.window_name('TaurusPlot (qwt5)')
```

Comment:
This window name is already wrong in the develop branch. It should be `TaurusTrend (qwt5)` instead of `TaurusPlot (qwt5)`